content/en/docs/contribute/start.md

```diff
@@ -217,7 +217,7 @@ to base your work on. Use these guidelines to make the decision:
     to beta in the next minor version, you need to know what the next minor
     version number is.
   - Find the release branch named for that version. For example, features that
-    changed in the v{{< release-branch >}} release got documented in the branch
+    changed in the {{< release-branch >}} version got documented in the branch
     named `dev-{{< release-branch >}}`.
```

Comment:
I don't think there are any shortcodes that work exactly how things are needed to here, so the easy fix looks like:
```suggestion
    changed in Kubernetes version 1.16 got documented in the branch named
    `dev-1.16`, which was then merged into `master` as part of the release process.
```

Reply:
Does that make sense @parthjawale ?

Reply:
Yes, sounds perfect.
I'm gonna commit this to my PR.